Box bbox
stereo_msgs/DisparityImage disp_img
---
float32 distance